doc/client_lua_api.md

```diff
@@ -131,7 +131,7 @@ The `:` prefix can also be used for maintaining backwards compatibility.
 
 Sounds
 ------
-**NOTE: Not fully implemented yet.**
+**NOTE: max_hear_distance and connecting to objects is not implemented.**
```

Comment:
Reading the code I don't see it parse `object` so I assume that it doesn't allow passing any object ref at all.

which is fine, but it's not what I was asking about above.